libctru/include/3ds/svc.h

```diff
@@ -994,7 +994,7 @@ void svcBreak(UserBreakType breakReason);
  * @param croInfo Library information.
  * @param croInfoSize Size of the above structure.
  */
-void svcBreakRO(UserBreakType breakReason, const void* croInfo, u32 croInfoSize);
+void svcBreakRO(UserBreakType breakReason, const void* croInfo, u32 croInfoSize) __asm__("svcBreak");
```

Comment:
The syscall is not called svcBreakRO and the extra parameters do not exist (not even dummy NULLs) for non-RO break reasons (syscall ABI is *not* ARM-EABI).